docs/configurations.md

```diff
@@ -25,6 +25,7 @@ This list contains configuration variables that can be used with all balena devi
 | BALENA_HOST_DISCOVERABILITY | boolean | false | true | Enable / Disable Avahi to run which will allow the device to respond to requests such as network scans. | v11.9.2 |
 | BALENA_HOST_SPLASH_IMAGE | integer | true | /boot/splash/balena-logo-default.png | Allows changing splash screen on boot to user defined file from userspace. | v12.3.0 |
 | BALENA_SUPERVISOR_HARDWARE_METRICS | boolean | false | true | Toggle hardware metrics reporting to the cloud, which occurs every 10 seconds when there are changes. Metrics include CPU utilization, CPU temperature, memory usage, and disk space. Useful for devices with bandwidth sensitivity. | v12.8.0 |
+| BALENA_SUPERVISOR_PATCH_INTERVAL | integer | false | 60000 | Define the frequency of updating the cloud with device's statel in milliseconds. The minimum value for this variable is defined by the balenaCloud backend, and may vary. | v12.12.0 |
```

Comment:
Can we call this `BALENA_SUPERVISOR_REPORT_INTERVAL`? Patch seems too specific. Also I'm wondering where did you get 60s? It used to be 10s?

Reply:
Sorry, just saw your comment. Ignore the part about the interval.